src/components/Buttons/TertiaryButton.story.tsx

```diff
@@ -19,6 +20,8 @@ storiesOf('Buttons', module)
       TertiaryButtonDisplayStyle.Dark
```

Comment:
TertiaryButton... I'm starting to think that maybe this naming convention will not scale very well =/

Reply:
Agreed. But I don't think we should have more than 3. Anyways, I will leave this for the future decisions coming from the Design team. Nothing set in stone yet.

Reply:
lgtm